.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.composer/vendor
```

Comment:
Only the composer cache is stored, not project vendor.

This is installed on sonata projects and works well.

For rarely cases you have issue with composer cache, you can clean it directly from Travis.